src/test/java/wc3libs/bin/ExeVersionTest.java

```diff
@@ -0,0 +1,66 @@
+package wc3libs.bin;
+
+import net.moonlightflower.wc3libs.misc.exeversion.ExeVersionPe;
+import net.moonlightflower.wc3libs.misc.exeversion.ExeVersionWmic;
+import net.moonlightflower.wc3libs.misc.exeversion.VersionExtractionException;
+import net.moonlightflower.wc3libs.port.Orient;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+import java.nio.file.Paths;
+
+import static org.testng.Assert.*;
+
+public class ExeVersionTest {
+    private static final Logger log = LoggerFactory.getLogger(ExeVersionTest.class.getName());
+
+    @Test()
+    public void extractExeVersionTest() throws VersionExtractionException, Exception {
+        // These tests need some static .exe files we don't carry around currently.
```

Comment:
I think we should rather ship some .exe file in test resources, then it can also work on CircleCI

Reply:
However not the WMIC part?.. The CI runs on Linux (or whatever but not Windows). I'm gonna look to create a small program with PE Version info.

Reply:
Hi, any update on this? You can disable the wmic part of course.

Reply:
Yes still "in progress". At first I wanted to create a cross-compiled executable from Linux, so replicatable builds would be easy. But I don't see any way at all, nor any tools, to modify Windows-specific PE data from Linux. I just finished cleaning up my workspace today, I will follow-up soon and make a simple static binary for tests. Cheers